features/admin.connections.v1/resources/guides/swe/quick-start.tsx

```diff
@@ -257,102 +258,17 @@ const SIWEAuthenticationProviderQuickStart: FunctionComponent<SIWEAuthentication
             </Grid>
             {
                 showApplicationModal && (
-                    <Modal
-                        data-testid={ componentId }
-                        open={ true }
-                        className="wizard application-create-wizard"
-                        dimmer="blurring"
-                        size="large"
+                    <ApplicationSelectionModal
+                        data-testid={ `${ componentId }-application-selection-modal` }
```

Comment:
If this is a new component shall we change `data-testid` to `data-componentid` ?

Reply:
Nope. this is an existing component. Despite the presence of such a component, a custom implementation has been done in SIWE and google quick start pages. This PR replaces that custom impl. with the existing `ApplicationSelectionModal` component.

Hence, changing `data-testid` to `data-componentid` of that existing component would be out of scope of this PR.